include/libssh2.h

```diff
@@ -926,6 +926,10 @@ LIBSSH2_API void libssh2_session_set_timeout(LIBSSH2_SESSION* session,
                                              long timeout);
 LIBSSH2_API long libssh2_session_get_timeout(LIBSSH2_SESSION* session);
 
+LIBSSH2_API void libssh2_session_set_read_timeout(LIBSSH2_SESSION* session,
+                                             long timeout);
```

Comment:
It'd be nice to indent the continued line.

Reply:
Oops, thank you for spotting that.